docs/topics/rn-known-issues.adoc

```diff
@@ -16,48 +16,23 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUPRULE-598[WINDUPRULE-598]
-|VS Code Extension
-|Quick Fix does not fix XML rules.
-
-|link:https://issues.redhat.com/browse/WINDUPRULE-589[WINDUPRULE-589]
-|VS Code Extension
-|Right-clicking a VS Code configuration and selecting *Open* does not always open the configuration.
-
-|link:https://issues.redhat.com/browse/WINDUPRULE-516[WINDUPRULE-516]
-|VS Code Extension
-|Quick Fix does not fix multiple rules.
-
-|link:https://issues.redhat.com/browse/WINDUP-2709[WINDUP-2709]
-|Eclipse Plugin
-|Installing the Eclipse plugin 5.0.0 on CodeReady Studio 12.16 requires Maven.
-
-|link:https://issues.redhat.com/browse/WINDUP-2692[WINDUP-2692]
-|Web UI & Windup-as-a-Service
-|When RHAMT 4.3.1 is deployed on OpenShift Container Platform with the `deploy.sh` script, an invalid route is created in the namespace.
+|link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
+|Eclipse Che plugin
+|MTA plugin does not work with CodeReady Studio version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-2645[WINDUP-2645]
-|Eclipse Che Plugin
-|Quick Fix does not fix multiple rules.
+|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
+|Migration-Application
+|The bash completion feature does not work.
```

Comment:
"Bash" is a command language, so  you should capitalize it. You can check Wikipedia